composer.json

```diff
@@ -62,7 +62,10 @@
     "autoload-dev": {
         "psr-0": {
             "": "engine/tests/phpunit/"
-        }
+        },
+		"files": [
+			"engine/tests/phpunit/bootstrap.php"
```

Comment:
We still need to require bootstrap somehow/somewhere to not break tests pending in PRs or already in master.